Hm/Relation.lean

```diff
@@ -77,4 +77,22 @@ def comp : Relation α γ := { t | ∃ a, (t.fst, a) ∈ R ∧ (a, t.snd) ∈ S}
 
 infixr:60 "∘" => comp
 
+theorem comp_comm {R₁ R₂ : Set α} : (∃ x, x ∈ R₁ ∧ x ∈ R₂) → ∃ x, x ∈ R₂ ∧ x ∈ R₁ := by
```

Comment:
This does not prove comp_comm (it can't, after all relation composition isnt commutative)